### PR TITLE
Version2

### PR DESCRIPTION
**Pull request by kalisjoshua**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Contributor Covenant Code of Conduct
```
Comment (coyotesqrl) [inline CODE_OF_CONDUCT.md:1]: No comment on content (I've used this same covenant before as well), but a comment that repo meta-files like this, the contributor's guide, and any templates (PR, Issue, etc.) that get created might also reside in a `.github` directory. It's a matter of preference; I prefer to minimize root folder clutter, but others don't.

So this is just a suggestion based on my preference that you may or may not share.

File: README.md [hunk 2]
```diff
@@ -77,91 +49,64 @@ beforehand, to ascertain that the RFC may be desirable; having a consistent
...
-
-
-### Writing an RFC
-
-RFCs should be written in a manner that it will reflect the final design of the
-RFC. However, given that the nature of the process means that the final
-implementation of the RFC - at the time of the next release - cannot be known
-when writing the RFC, modifications to "active" RFCs can be done in follow-up
-pull requests but should be the exception and not the rule.
+was made; try commenting on the existing pull request before opening a new RFC
```
Comment (coyotesqrl) [inline README.md]: Missing terminal period.

**Review by coyotesqrl — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (kalisjoshua) [reply CODE_OF_CONDUCT.md:1]: I like the idea @coyotesqrl but I also like the idea of them being as visible as possible (in the root) as possible to help with those people who don't read the email; having them in the root means there is an additional opportunity for people to see them.

**Comment by AmeguidGRate** [comment]
ping

**Comment by AmeguidGRate** [comment]
image

**Pull request merged**